istioctl/pkg/validate/validate.go

```diff
@@ -158,6 +163,39 @@ func (v *validator) validateResource(istioNamespace string, un *unstructured.Uns
 		v.validateDeploymentLabel(istioNamespace, un)
 		return nil
 	}
+
+	if un.GetAPIVersion() == "install.istio.io/v1alpha1" {
+		if un.GetKind() == "IstioOperator" {
+			if err := checkFields(un); err != nil {
+				return err
+			}
+
+			// IstioOperator isn't part of pkg/config/schema/collections,
+			// usual conversion not available.
+
+			spec, ok := un.Object["spec"]
+			if !ok {
+				return fmt.Errorf("IstioOperator lacks spec") // nolint: golint,stylecheck
```

Comment:
nit: probably better to avoid annotations, you could work around this with something like "spec field missing in IstioOperator".